thoth/storages/graph/postgres.py

```diff
@@ -406,6 +406,12 @@ def get_alembic_version_count_all(self) -> int:
             result = session.execute(query).fetchone()
             return result[0]
 
+    def get_last_solver_datetime(self) -> datetime:
```

Comment:
do we also need solver filters (os_name, os_version, python_version)? or only generic latest solver run?

Reply:
That's a good question. Given the issue https://github.com/thoth-station/metrics-exporter/issues/822 I understood this was for all solver documents, @fridex wdyt?

Reply:
actually, can't we reuse this query which has all filters already https://github.com/thoth-station/storages/blob/9bbc64b06c11efe7f85f616aea76be5db6f326f8/thoth/storages/graph/postgres.py#L2540?

Reply:
I agree, this query has a parameter to sort the documents in datetime order (latest first), so it would be more convenient :+1:

Reply:
> do we also need solver filters (os_name, os_version, python_version)? or only generic latest solver run?

We wanted to expose datetime in dashboards so we are aware "how old the database is". Additional filters can be introduced to the method for completeness, but I don't know if that's something we want to show on dashboards. Only a generic one could be good enough.

> actually, can't we reuse this query which has all filters already

The method is supposed to get the latest datetime for solvers. The referenced one will get solver ids (which might require an additional query to get the actual datetime).

Reply:
> The method is supposed to get the latest datetime for solvers.

Right, then it's better to leave it as a different query :+1: 
Concerning the filters, I will let the query as it is if it's just a matter of showing the latest update of the database

Reply:
> I will let the query as it is if it's just a matter of showing the latest update of the database

Up to you. I can imagine a more detailed dashboard overview showing datetime for each solver registered and freshness of the dependency data it computes - that might uncover issues that certain runtime environment is not updating (ex. some deployment issue).

Reply:
I will leave this as it is for the moment, and we can open a separate issue to improve the dashboard with more details if that's ok with you :+1: